src/auxiliary/special_elixirs.jl

```diff
@@ -36,6 +36,16 @@ julia> redirect_stdout(devnull) do
 ```
 """
 function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
+    # Check that all kwargs exist as assignments
+    code = read(elixir, String)
+    expr = Meta.parse("begin \n$code \nend")
+    expr = insert_maxiters(expr)
+
+    for (key, val) in kwargs
+        # This will throw an error when `key` is not found
+        find_assignment(expr, key)
+    end
```

Comment:
What I don't understand here is that you cann `insert_maxiters` before calling `find_assignment`. It looks like this doesn't work, does it?

Reply:
The problem is that `insert_maxiters` only inserts `maxiters` if the elixir has a call to `solve`. The restart elixirs don't have that, so no `maxiters` is inserted. But `@test_trixi_include` still tries to call it with `maxiters=1` when running coverage tests.

Reply:
I see! We should change that so that we also insert `maxiters` into `init`, shouldn't we?

Reply:
Do you use other `init` functions anywhere? I'm not sure what will be the best approach: having to add `maxiters` to all `init` calls manually or dealing with the potential problem that there might be another `init` function in an elixir. Any thoughts, @sloede?

Reply:
If this really only concerns `init` functions in restart files, I'd probably just add the `maxiters` to them. There are probably less than ten elixirs in Trixi.jl that do restarts, thus we can avoid making a hard decision right now.

Reply:
There are exactly 7 such elixirs, and I've added `maxiters` to all of them in this PR.

Reply:
Okay, let's do it like this.